src/tests/work-item/work-item-list/work-item-list.spec.js

```diff
@@ -65,7 +65,7 @@ describe('Work item list', function () {
       expect(page.workItemDescription(page.workItemByNumber(0))).toBe('Some Description');
       expect(page.workItemTitle(page.workItemByNumber(0))).toBe('Some Title');
       page.allWorkItems.getText().then(function (text) {
-        expect(text).toContain("View Details Delete\nnew\n1\nSome Title\nSome Description");
+        expect(text).toContain("new\n1\nSome Title 1\nSome Description 1\nView Details Delete");
```

Comment:
Works.
Just as a general comment: I am not sure that is the recommended way of doing that. getText() "flattens" the text in the element including all childs. This might work (it's flaky in my opinion), but it essentially checks the hierarchy of the childs. If we decide to put the details button before the delete button for example, this would break.